tests/test_inventory.py

```diff
@@ -211,7 +211,27 @@ def test_activities(self) -> None:
         self.assertEqual(inv.activities(), {"I-123": 5.8, "Tc-99m": 2.3})
         inv = Inventory({"Tc-99m": 2300, "I-123": 5800})
         self.assertEqual(inv.activities("kBq"), {"I-123": 5.8, "Tc-99m": 2.3})
+    
+    def test_energies(self) -> None:
+        """
+        Test Inventory.energies() method.
+        """
+
+        inv = Inventory({"H-3": 1})
+        eng = inv.energies()
+        self.assertEqual(list(eng.keys()), ["H-3"])
+        dict_assert_almost_equal(self, eng["H-3"], {"B-": 18.59202})
+
+        inv = Inventory({"Pa-234m": 1})
+        eng = inv.energies()
+        self.assertEqual(list(eng.keys()), ["Pa-234m"])
+        dict_assert_almost_equal(
+            self, eng["Pa-234m"], {"B-": 2190.40024, "IT": 0.118272}
```

Comment:
For the B- decay modes, it's outputting the total Q_decay value

I'm not sure this is actually what most end users would need however, as that includes the energy carried away by the antineutrino.

E.g. is someone is interested in finding the decay heat of an inventory, or approximating the radiation dose, you wouldn't usually include the energy carried away by neurtinos in the calculation as that won't be deposited on earth (but some galaxy far away)

So I guess my preference would be that we don't include that in the output by default, and document the behaviour clearly so users understand what `.energies()` is returning

Reply:
Can you explain a bit more? I'm very new to radioactivity in general so there's much I dont understand.  Are you saying that most beta decay does not actually result in an energy transfer to the immediate area?  What percent/value should be subtracted to account for the antineutrino?  

Maybe this will be covered by/built in to the new data I need to bring in but would still love to understand this more if possible